<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_aceptar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SI</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_decision">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_salir">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>¿Esta seguro de enviar la Decisión?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
